src/lib/hooks.ts

```diff
@@ -938,3 +939,9 @@ export function useMetadata(
 
   return { allMetadataLoaded, data, getMetadata };
 }
+
+export const useGridSizeState = () => {
+  return useLocalStorage("large-grid", {
```

Comment:
```suggestion
  return useLocalStorage("mp:large-grid", {
```

Just a quick edit to add a prefix.